kubernetes/server/create.sh

```diff
@@ -33,7 +33,7 @@ spec:
     spec:
       containers:
       - name: scdf
-        image: springcloud/spring-cloud-dataflow-server-kubernetes:latest
+        image: springcloud/spring-cloud-dataflow-server-kubernetes:$DATAFLOW_VERSION
```

Comment:
For docker images we publish milestone, release candidate and release artifacts in the typical format (https://hub.docker.com/r/springcloud/spring-cloud-dataflow-server-kubernetes/tags/) but we don't publish explicit SNAPSHOT versions. When the default are used, an image by name of`1.5.0.BUILD-SNAPSHOT` is attempted to be fetched which doesn't exist. When master is built in CI we publish to a docker tag `latest` which represents the current master "SNAPSHOT".

I think in this case we want to check the `$DATAFLOW_VERSION` and if its a `SNAPSHOT` change the tag to `latest`. We do a similar manipulation in the documentation where we align image names to versions: https://github.com/spring-cloud/spring-cloud-dataflow/blob/master/spring-cloud-dataflow-docs/pom.xml#L325